src/import/FSHTank.ts

```diff
@@ -179,6 +203,41 @@ export class FSHTank implements Fishable {
               e.id === item ||
               getUrlFromFshDefinition(e, this.config.canonical) === item
           );
+          if (!result) {
+            // There may be a matching definitional Instance of StructureDefinition with type Extension
+            result = this.getAllInstances().find(
+              extensionInstance =>
+                (extensionInstance.name === item ||
+                  extensionInstance.id === item ||
+                  getUrlFromFshDefinition(extensionInstance, this.config.canonical) === item) &&
+                extensionInstance.instanceOf === 'StructureDefinition' &&
+                extensionInstance.usage === 'Definition' &&
+                extensionInstance.rules.some(
+                  rule =>
+                    rule instanceof AssignmentRule &&
+                    rule.path === 'derivation' &&
+                    rule.value instanceof FshCode &&
+                    rule.value.code === 'constraint'
+                ) &&
+                extensionInstance.rules.some(
+                  rule =>
+                    rule instanceof AssignmentRule &&
+                    rule.path === 'type' &&
+                    rule.value === 'Extension'
+                )
+            );
+          }
+          if (!result) {
+            // There may be a matching inline Instance of Extension
+            result = this.getAllInstances().find(
+              extensionInstance =>
+                (extensionInstance.name === item ||
+                  extensionInstance.id === item ||
+                  getUrlFromFshDefinition(extensionInstance, this.config.canonical) === item) &&
+                extensionInstance.instanceOf === 'Extension' &&
+                extensionInstance.usage === 'Inline'
+            );
+          }
           break;
         case Type.Logical:
```

Comment:
Why exclude `Logical` and `Resource` here? In theory they would work similarly to `Profile` and `Extension` right?

Reply:
I think they would work similarly, yes. I didn't include them because they weren't listed in the task description. I think we're in the process of permitting `InstanceOf` a given `Logical`... but that's different than being an `InstanceOf` `StructureDefinition`, with a similar sort of reasoning as for my incorrect implementation for handling `InstanceOf` `Extension`. So I'll defer to @cmoesel on this, since he created the task.

Reply:
I think this was an oversight on my part.  It makes sense to support instances of logicals and resources too.  I don't know how often it will come up, but if it does, we should support it.  It seems it would be inconsistent not to support it.